grails-app/controllers/org/pih/warehouse/api/LoadDataApiController.groovy

```diff
@@ -13,10 +13,41 @@ import grails.converters.JSON
 
 class LoadDataApiController extends BaseDomainApiController {
 
+    def loadDataService;
+
     def listOfDemoData = {
         def listOfDemoData = grailsApplication.config.openboxes.configurationWizard.listOfDemoData
         
         render([data: listOfDemoData] as JSON)
     }
 
+
+
+    def load = {
+        loadDataService.importOrganizations(
+                new URL(grailsApplication.config.openboxes.configurationWizard.dataFiles.organizations)
```

Comment:
@hdsldv Perhaps it would be worth of adding the checks if the specific data loader is enabled or at least present in the config. So either:
```
if (grailsApplication.config.openboxes.demoData.organizations) {
    loadDataService.importOrganizations(new URL(grailsApplication.config.openboxes.demoData.organizations)
}
```
or url could be moved to the `url` parameter and we could add `enabled` prop:
```
if (grailsApplication.config.openboxes.demoData.organizations.enabled) {
    loadDataService.importOrganizations(new URL(grailsApplication.config.openboxes.demoData.organizations.url)
}
```
We could also include the token indexes for specific properties if we need. For example:
```
grailsApplication.config.openboxes.demoData = [
    organizations: [
        enabled: true.
        url: "urlToFile",
        nameIndex: 1, // 1 by default
        typeIndex: 2, // 2 by default
    ],
    (...)
]
```

cc @pmuchowski @jmiranda

Reply:
Added enabling via config, column value are now accessed via names